ballerina-tests/tests/45_services_with_headers_service.bal

```diff
@@ -22,27 +22,27 @@ listener grpc:Listener ep45 = new (9145);
 @grpc:ServiceDescriptor {descriptor: ROOT_DESCRIPTOR_45_SERVICES_WITH_HEADERS, descMap: getDescriptorMap45ServicesWithHeaders()}
 service "HeadersService" on ep45 {
 
-    remote function unary(HeadersServiceHSResCaller caller, ContextHSReq req) returns error? {
+    remote function unary(HeadersServiceHSResCaller caller, ContextHSReq req) {
         map<string|string[]> responseHeaders = {};
         if !req.headers.hasKey("unary-req-header") {
-            check caller->sendError(error grpc:AbortedError("unary-req-header header is missing"));
+            checkpanic caller->sendError(error grpc:AbortedError("unary-req-header header is missing"));
             return;
         } else {
-            string headerValue = check grpc:getHeader(req.headers, "unary-req-header");
+            string headerValue = checkpanic grpc:getHeader(req.headers, "unary-req-header");
             log:printInfo("Request Header: " + headerValue);
             responseHeaders["unary-res-header"] = ["abcde", "fgh"];
         }
-        check caller->sendContextHSRes({content: req.content, headers: responseHeaders});
-        check caller->complete();
+        checkpanic caller->sendContextHSRes({content: req.content, headers: responseHeaders});
+        checkpanic caller->complete();
     }
 
-    remote function serverStr(HeadersServiceHSResCaller caller, ContextHSReq req) returns error? {
+    remote function serverStr(HeadersServiceHSResCaller caller, ContextHSReq req) {
         map<string|string[]> responseHeaders = {};
         if !req.headers.hasKey("server-steaming-req-header") {
-            check caller->sendError(error grpc:AbortedError("server-steaming-req-header header is missing"));
+            checkpanic caller->sendError(error grpc:AbortedError("server-steaming-req-header header is missing"));
             return;
         } else {
-            string headerValue = check grpc:getHeader(req.headers, "server-steaming-req-header");
+            string headerValue = checkpanic grpc:getHeader(req.headers, "server-steaming-req-header");
```

Comment:
```suggestion
            string headerValue = check grpc:getHeader(req.headers, "server-steaming-req-header");
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/c32ef7e98e2bfaa4df966a96025d16d755dfb1cd